sapi/phpdbg/phpdbg_opcode.c

```diff
@@ -114,6 +114,44 @@ char *phpdbg_decode_opline(zend_op_array *ops, zend_op *op) /*{{{ */
 		break;
 	}
 
+#if 1
```

Comment:
okay, then please turn it to #if 0. It shouldn't be that useful, except when working on exactly this.